lib/template/lib/routes.rb

```diff
@@ -4,7 +4,7 @@
   use Pliny::Middleware::CORS
   use Pliny::Middleware::RequestID
   use Pliny::Middleware::RequestStore, store: Pliny::RequestStore
-  use Pliny::Middleware::Timeout, timeout: Config.timeout if Config.timeout > 0
+  use Rack::Timeout if Config.timeout > 0
```

Comment:
But isn't the app config outside the Pliny namespace anyways? eg: https://github.com/interagent/pliny/blob/master/lib/template/config/config.rb